c/MemSafety-LinkedLists.set

```diff
@@ -8,4 +8,6 @@ ddv-machzwd/*_true-valid-memsafety*.i
 ddv-machzwd/*_false-valid-memtrack*.i
 forester-heap/*_true-valid-memsafety*.i
 forester-heap/*_false-valid-deref*.i
-
+list-ext3-properties/*_true-valid-memsafety*.i
+list-ext3-properties/*_false-valid-deref*.i
+list-ext3-properties/*_false-valid-memtrack*.i
```

Comment:
@emankcin: Here, the line `list-ext3-properties/*_false-valid-memcleanup*.i` is missing, otherwise we would fail to include 4 of the 16 new tasks!

Reply:
I missed that completely, thank you!